### PR TITLE
Webtestclient support

### PR DESCRIPTION
**Pull request by OlgaMaciaszek**

### CODE REVIEW

File: spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/WebTestClientJUnitMethodBodyBuilder.groovy [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+@PackageScope
+class WebTestClientJUnitMethodBodyBuilder extends RestAssuredJUnitMethodBodyBuilder {
+
+    WebTestClientJUnitMethodBodyBuilder(Contract stubDefinition, ContractVerifierConfigProperties configProperties) {
+        super(stubDefinition, configProperties)
+    }
+
+    @Override
+    protected String returnedResponseType() {
+        return 'WebTestClientResponse'
```
Comment (marcingrzejszczak) [inline spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/WebTestClientJUnitMethodBodyBuilder.groovy]: Why, oh why do you use single `'` ? :( :( :(

File: spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/util/SyntaxChecker.groovy [hunk 2]
```diff
@@ -106,15 +118,22 @@ class SyntaxChecker {
...
 		sourceCode.append(test)
 		return new GroovyShell(SyntaxChecker.classLoader, configuration).parse(sourceCode.toString())
 	}
 
-	static Class tryToCompileJava(String test) {
+	private static GString getStaticImports(String builderName) {
+		if (builderName.toLowerCase().contains('webtestclient')) {
+			return "$WEB_TEST_CLIENT_STATIC_IMPORTS\n"
+		}
+		"$STATIC_IMPORTS\n"
```
Comment (marcingrzejszczak) [inline spring-cloud-contract-verifier/src/test/groovy/org/springframework/cloud/contract/verifier/util/SyntaxChecker.groovy]: `return` is missing and the method should return `String`

File: spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/WebTestClientJUnitMethodBodyBuilder.groovy (hunk 1 above)
Comment (OlgaMaciaszek) [reply spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/WebTestClientJUnitMethodBodyBuilder.groovy]: Cause I like it.

Comment (OlgaMaciaszek) [reply spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/builder/WebTestClientJUnitMethodBodyBuilder.groovy]: And cause we had a crazy mixture of both.

**Pull request merged**